Dockerfile

```diff
@@ -39,7 +39,7 @@ LABEL maintainer="IBM Quality Measure and Cohort Service Team" \
       description="Quality Measure and Cohort Service" \
       name="cohorting-app" \
       vendor="Alvearie Open Source by IBM" \
-      version="1.0" \
+      version="2.0.1" \
```

Comment:
Is this set to 2.0.1 because it is what is in development for release after we post 2.0.0 (e.g. you will set the version to 2.0.0 on the release branch)? My first thought when looking at it was that we were skipping 2.0.0 and going directly to 2.0.1 which was weird.

Reply:
Correct. When I cut the release branch, the version for that will be 2.0.0 in the release branch. I wanted to remove the sonar file in both branches so just did the version update here to avoid having another PR.